File: back/src/main/java/com/realive/dto/order/OrderCancelRequestDTO.java
package com.realive.dto.order;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class OrderCancelRequestDTO {
    private Long orderId;
    private Long customerId;
    private String reason; // 취소 사유
}

File: RealFE/src/types/orders/orderCancelRequestDTO.ts

export interface OrderCancelRequestDTO {
    orderId: number;    // Spring의 Long은 TypeScript에서 number로 매핑됩니다.
    customerId: number; // Spring의 Long은 TypeScript에서 number로 매핑됩니다.
    reason: string;     // 취소 사유
}